src/tests/error.rs

```diff
@@ -9,10 +9,6 @@ test_case!(error, async move {
             TranslateError::UnsupportedStatement("TRUNCATE TABLE TableA".to_owned()).into(),
             "TRUNCATE TABLE TableA;",
         ),
-        (
-            TranslateError::UnsupportedUnaryOperator("!".to_owned()).into(),
-            "SELECT 2! FROM TableA;",
-        ),
```

Comment:
Now, it looks that we support all unary operator which `sqlparser-rs` supports.
Let's rename the error.
```rust
#[error("unreachable unary operator: {0}")]
UnreachableUnaryOperator(String),          
```

I saw the parser code, because of the `PostgreSqlDialect`, `Unreachable-` prefix in our code looks inevitable.
![image](https://user-images.githubusercontent.com/2025065/141667657-75bf3658-189f-4bd1-ae3b-e42c98745332.png)
https://github.com/sqlparser-rs/sqlparser-rs/blob/9c87d3b195ecc05f92927197e29f5285c3aa5e9b/src/parser.rs#L416-L434